src/java_bytecode/java_class_loader.h

```diff
@@ -96,6 +101,7 @@ class java_class_loadert:public messaget
 private:
   std::map<std::string, jar_filet> m_archives;
   std::vector<irep_idt> java_load_classes;
+  bool throw_runtime_exceptions;
```

Comment:
I am not a big fan of uninitialized Boolean variables, having been bitten by difficult to track-down Heisenbugs. Would you mind giving a default value for this and the above?

Reply:
I gave both of them `false` as a default value. (Seemed like the safer option as the test-gen `--java-throw-runtime-exceptions` parameter isn't/can't be enabled on platform yet.)